test/Conversion/HWToLLVM/convert_aggregates.mlir

```diff
@@ -108,5 +108,8 @@ func.func @convertStruct(%arg0 : i32, %arg1: !hw.struct<foo: i32, bar: i8>) {
   // CHECK: llvm.insertvalue %arg0, %[[SCAST1]][1] : !llvm.struct<(i8, i32)>
   %1 = hw.struct_inject %arg1["foo"], %arg0 : !hw.struct<foo: i32, bar: i8>
 
+  // CHECK-NEXT: %[[SCAST2:.*]] = llvm.extractvalue %[[SCAST]][1] : !llvm.struct<(i8, i32)>
+  // CHECK-NEXT: %[[SCAST3:.*]] = llvm.extractvalue %[[SCAST]][0] : !llvm.struct<(i8, i32)>
+  %2:2 = hw.struct_explode %arg1 : !hw.struct<foo: i32, bar: i8>
```

Comment:
Could you add a test for `hw.struct_explode %emptystruct : !hw.struct<>` to be sure?

Reply:
Did you maybe add `%0 = hw.struct_explode %emptystruct : !hw.struct<>` (i.e., define a result value where no result value should be)?

Reply:
Sorry for the late reply because of the Spring Festival. I misunderstanded your meanings:cry:, so I add this test, although there is no generated result for it, the test case was tested and has no problem. Thank for your review! :heart:

Reply:
If there is no problem, help me merge it.

Reply:
Thanks! Yes, that's the expected behavior. I just wanted to make sure that nothing bad will happen for this corner case.